### PR TITLE
UI now showing broker list when DISABLELOGDIRSCOLLECTION=true

### PR DESCRIPTION
**Pull request by kamsharipova**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [ ] Manually (please, describe, if necessary)
- [x] Unit checks
- [x] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/Brokers/BrokersList/BrokersList.tsx [hunk 1]
```diff
@@ -29,14 +29,26 @@ const BrokersList: React.FC = () => {
   } = clusterStats;
 
   const rows = React.useMemo(() => {
-    if (!diskUsage) return [];
+    let brokersResource;
+    if (!diskUsage) {
+      brokersResource =
+        brokers?.map(({ id, host, port }) => {
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/Brokers/BrokersList/BrokersList.tsx:35]: please check the Linter warnings

File: kafka-ui-react-app/src/components/Brokers/BrokersList/__test__/BrokersList.spec.tsx [hunk 2]
```diff
@@ -134,12 +134,21 @@ describe('BrokersList Component', () => {
...
         renderComponent();
         expect(screen.getByRole('table')).toBeInTheDocument();
-        expect(
-          screen.getByRole('row', { name: 'Disk usage data not available' })
-        ).toBeInTheDocument();
+        expect(screen.getAllByRole('row').length).toEqual(3);
+      });
+      it('opens broker when row clicked', async () => {
+        renderComponent();
+        await act(() => {
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/Brokers/BrokersList/__test__/BrokersList.spec.tsx:144]: i think there with the new version of user event , you can automatically await the userEvent

**Review by Mgrdich — CHANGES_REQUESTED** [review]

File: kafka-ui-react-app/src/components/Brokers/BrokersList/__test__/BrokersList.spec.tsx (hunk 2 above)
Comment (kamsharipova) [reply kafka-ui-react-app/src/components/Brokers/BrokersList/__test__/BrokersList.spec.tsx]: fixed

File: kafka-ui-react-app/src/components/Brokers/BrokersList/BrokersList.tsx (hunk 1 above)
Comment (kamsharipova) [reply kafka-ui-react-app/src/components/Brokers/BrokersList/BrokersList.tsx]: fixed

File: kafka-ui-react-app/src/components/Brokers/BrokersList/__test__/BrokersList.spec.tsx [hunk 3]
```diff
@@ -134,12 +134,20 @@ describe('BrokersList Component', () => {
         }));
       });
 
-      it('renders empty table', async () => {
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/Brokers/BrokersList/__test__/BrokersList.spec.tsx:137]: why are we deleting the empty table case ?

Comment (kamsharipova) [reply kafka-ui-react-app/src/components/Brokers/BrokersList/__test__/BrokersList.spec.tsx:137]: fixed

**Review by Mgrdich — APPROVED** [review]

**Comment by iliax** [comment]
checked with docker-compose with `KAFKA_CLUSTERS_0_DISABLELOGDIRSCOLLECTION: "true"` set 
broker list is still empty:

Screenshot 2022-12-07 at 15 34 07

,

Screenshot 2022-12-07 at 15 34 15

**Comment by iliax** [comment]
verified, working fine now 
Screenshot 2022-12-07 at 17 10 40

**Review by iliax — APPROVED** [review]

**Comment by Haarolean** [comment]
@kamsharipova thank you <3

**Pull request merged**